kuma/wiki/kumascript.py

```diff
@@ -88,7 +96,9 @@ def _get_attachment_metadata_dict(attachment):
 # be renamed to render_document(), and the post() method above should
 # be renamed to render_string(), maybe. For now, though, there are so
 # many tests that mock kumascript.get() that I've left the name unchanged.
-def get(document, base_url, timeout=config.KUMASCRIPT_TIMEOUT):
+def get(document, base_url,
+        cache_control='',
```

Comment:
Super minor, but it's more common practice in this case to use `cache_control=None`.